.env.example

```
@@ -23,6 +23,16 @@ HTML_CACHE_REMOVE_SCRIPTS=true
 HTML_CACHE_REMOVE_STYLES=false    # Keep CSS
 HTML_CACHE_REMOVE_COMMENTS=false  # Keep comments
 
+# HTML Cache TTL Configuration (values in minutes)
+HTML_CACHE_TTL_LIVE=30         # Live data: start list (before race), stages (during race)
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)